app/src/editor/canvas/components/ModuleSearch.pcss

```diff
@@ -83,6 +95,31 @@
   color: #CDCDCD;
 }
 
+.ModuleSearch .Input .ClearButton {
+  --iconSize: 14px;
+  --iconPadding: 3px; /* give additional hover leeway */
+
+  width: calc(2 * var(--iconPadding) + var(--iconSize));
+  height: calc(2 * var(--iconPadding) + var(--iconSize));
+  padding: var(--iconPadding);
+  line-height: 0;
+  flex-shrink: 0;
+  z-index: 1; /* ensure over resize control */
+}
+
+.ModuleSearch .Input .ClearButton svg {
+  transition: all 0.2s;
+}
+
+.ModuleSearch .Input .ClearButton:hover svg circle {
```

Comment:
`.ClearButton:hover circle` (selector) should be enough. I'd also consider converting the expand/collapse toggle into outlines and make them share that decoration.